tests/test_authenticator.py

```diff
@@ -1241,7 +1241,8 @@ def annotate_authentication_document(
             # authentication sub-documents are assembled properly and
             # placed in the right position.
             flows = doc['authentication']
-            oauth_doc, basic_doc = sorted(flows, key=lambda x: x['type'])
+            [oauth_doc] = list(filter(lambda x: isinstance(x, dict), flows))
```

Comment:
I'm not sure how to sort a mixed list that now contains lists and a dictionary, so I filtered by the type instead. I don't know if the order is now guaranteed as it was before.

Reply:
This is OK since there's only one item in each list.